src/ramsesitems.h

```diff
@@ -19,7 +19,7 @@
 #ifdef RAMSESLONGIDS
 #define RAMSESIDTYPE int
 #else
-#define RAMSESIDTYPE  unsigned int
+#define RAMSESIDTYPE int
```

Comment:
@JinsuRhee is it possible that this type could be different for different runs of RAMSES? Does it warrant wrapping this in idefs and adding some compilation options?